api/hypershift/v1beta1/zz_generated.featuregated-crd-manifests/hostedclusters.hypershift.openshift.io/AAA_ungated.yaml

```diff
@@ -3513,17 +3513,19 @@ spec:
                   This pull secret will be part of every payload generated by the controllers for any NodePool of the HostedCluster
                   and it will be injected into the container runtime of all NodePools.
                   Changing this value will trigger a rollout for all existing NodePools in the cluster.
-                  Changing the content of the secret inplace will not trigger a rollout and might result in unpredictable behaviour.
+                  Add the hypershift.openshift.io/reload=true label to the referenced pull secret to enable reloading of the content
+                  of the secret in-place.
                 properties:
                   name:
-                    default: ""
-                    description: |-
-                      Name of the referent.
-                      This field is effectively required, but due to backwards compatibility is
-                      allowed to be empty. Instances of this type with an empty value here are
```

Comment:
This is technically backwards incompat as well, unclear - would have been invalid anyway?

Reply:
(I couldn't embed `corev1.LocalObjectReference` as it somehow broke the applyconfiggenerator)